mne/viz/raw.py

```diff
@@ -185,11 +188,17 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
 
         .. versionchanged:: 0.18
            Support for ``filtorder=0`` to use FIR filtering.
-    clipping : str | None
+    clipping : str | float | None
         If None, channels are allowed to exceed their designated bounds in
         the plot. If "clamp", then values are clamped to the appropriate
         range for display, creating step-like artifacts. If "transparent",
         then excessive values are not shown, creating gaps in the traces.
+        If float, values are outside the float factor ``clipping`` beyond their
+        dedicated range, so ``clipping=1.`` is an alias for
+        ``clipping='transparent'``.
```

Comment:
I find this hard to understand. How about sth like:

```suggestion
        If float, clipping occurs for values exceeding their dedicated range by
        the factor ``clipping``. Consequently, ``clipping=1.`` is an alias for
        ``clipping='transparent'``.
```

However I can see that "exceeding X by factor N" would actually imply that, if `N=1`, we would _excced_ X by `1 X`, i.e., the value would be at `2 X`.

Yet this seems to be common language usage in the US, I suppose? E.g. tech companies always promote their products as being "4x faster" when actually the new speed is 4x the old speed, i.e. they're actually only 3 times faster, but _4 times as fast_.

Opinions?

Reply:
Pushing a commit to reword now